html/pfappserver/lib/pfappserver/Form/Violation.pm

```diff
@@ -176,12 +176,12 @@ has_field 'button_text' =>
 has_field 'vlan' =>
```

Comment:
Changing this mean that we have to change it every where in the code (db, violations.conf ...)